apps/back/src/Controller/DemoController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\DevTools\PHPStan\ThisRouteDoesntNeedAVoter;
+use App\Dto\Request\Demo\CreateDemoValidationDto;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpKernel\Attribute\MapRequestPayload;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+class DemoController extends AbstractController
```

Comment:
Maybe we can give a more understandable name to the Controller, like `ValidationDemoController` ?